composer.json

```diff
@@ -1,5 +1,19 @@
 {
     "require": {
-        "phan/phan": "^1.2"
+        "phan/phan": "^1.2",
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.7.0"
+    },
+    "require-dev": {
+        "drupal/coder": "^8.3.1",
+        "friendsofphp/php-cs-fixer": "^2.15",
+        "squizlabs/php_codesniffer": "^3.5"
+    },
+    "scripts": {
+        "check-coding-standards/php-cs-fixer": [
+            "php-cs-fixer --config=.php_cs.dist fix --dry-run"
+        ],
```

Comment:
We don't have `php-cs-fixer` installed. There should be `check-coding-standards/phpcs` and `apply-coding-standards/phpcs` entries (and possible generic `check-coding-standards`/`apply-coding-standards`) (cf. https://github.com/os2loop/profile/blob/main/composer.json).